generators/entity/index.js

```diff
@@ -386,7 +386,8 @@ class EntityGenerator extends BaseBlueprintGenerator {
         }
 
         this.entityConfig.name = this.entityConfig.name || context.name;
-        if (![SQL, MONGODB, COUCHBASE, NEO4J].includes(context.databaseType)) {
+        // disable pagination if there is not database, unless it is gateway project
```

Comment:
```suggestion
        // disable pagination if there is not database, unless it’s a microservice entity published by a  gateway
```

Reply:
Agree.  But if there are non-microservice entities  there ought be the database anyway?

Reply:
still, I don't recommend it thin GW with clear responsibility (routing + security) are more maintenable

Reply:
Anyway proposed change is implemented as proposed

Reply:
It’s missing to update the comment.

Reply:
pushed